src/signals/settings/users/containers/Overview/components/DataView/DataList/index.js

```diff
@@ -0,0 +1,56 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+
+import { StyledTR, StyledTD } from '../styled';
+
+const DataList = ({
+  columnOrder,
+  invisibleColumns,
+  data,
+  onItemClick,
+  primaryKeyColumn,
+}) => {
+  if (!data) return null;
+
+  const filterVisibleColumns = colHeader => invisibleColumns.includes(colHeader) === false;
+  const colHeaders = (columnOrder.length && columnOrder) || Object.keys(data[0]).filter(filterVisibleColumns);
+
+  return (
+    <Fragment>
+      {data.map((row, rowIndex) => (
+        <StyledTR
+          key={JSON.stringify(data[rowIndex])}
+          data-item-id={primaryKeyColumn && data[rowIndex][primaryKeyColumn]}
```

Comment:
To my knowledge, an attribute will not be set when its value is `null`. When `undefined` it's still set. If the attribute is not meant to be on the `<tr>`, then it would be better to use `data-item-id={primaryKeyColumn ? data[rowIndex][primaryKeyColumn] : null}`